Traceback (most recent call last):
  File "C:\Users\mirjam\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\mirjam\anaconda3\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\mirjam\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import pandas as pd
content_list = []
for newspaper in root.findall('.//ns0:TextRegion', ns):
	region = newspaper.get('id')
	if region in dict_order:
		groupvalues = dict_order[region]
		group = groupvalues[0][0]
		index = groupvalues[0][1]
	else:
		group = 0
		index = 0
	for content in newspaper.findall('.//ns0:Unicode', ns):
		content = content.text
	content_list.append([group, index, region, content])
newspaper_with_order = pd.DataFrame(content_list, columns = ["Group", "Index", "Region", "Content"])  
newspaper_with_order
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Input [1;32mIn [13][0m, in [0;36m<cell line: 3>[1;34m()[0m
[0;32m      1[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m
[0;32m      2[0m content_list [38;5;241m=[39m []
[1;32m----> 3[0m [38;5;28;01mfor[39;00m newspaper [38;5;129;01min[39;00m root[38;5;241m.[39mfindall([38;5;124m'[39m[38;5;124m.//ns0:TextRegion[39m[38;5;124m'[39m, [43mns[49m):
[0;32m      4[0m 	region [38;5;241m=[39m newspaper[38;5;241m.[39mget([38;5;124m'[39m[38;5;124mid[39m[38;5;124m'[39m)
[0;32m      5[0m 	[38;5;28;01mif[39;00m region [38;5;129;01min[39;00m dict_order:

[1;31mNameError[0m: name 'ns' is not defined
NameError: name 'ns' is not defined

